validator_client/src/config.rs

```diff
@@ -41,6 +42,11 @@ pub struct Config {
     pub graffiti: Option<Graffiti>,
     /// Graffiti file to load per validator graffitis.
     pub graffiti_file: Option<GraffitiFile>,
+    /// fee_recipient address, that will receive transaction fees collected from any blocks
+    /// produced by this node.
```

Comment:
```suggestion
    /// Fallback fallback address.
```

Just so it doesn't sound like the canonical source.